### PR TITLE
Fix preview UI

### PR DESCRIPTION
**Pull request by astorije**
See https://github.com/thelounge/thelounge/pull/2349

screen shot 2018-04-12 at 00 58 49

### CODE REVIEW

File: crypto.css [hunk 1]
```diff
@@ -109,5 +105,5 @@ a:hover {
 /* Previews */
 
 #chat .toggle-text {
```
Comment (xPaw) [inline crypto.css:107]: Is this rule necessary to keep?

Comment (astorije) [reply crypto.css:107]: Only if we keep the override L88. What do you think? Happy to remove both.

Comment (xPaw) [reply crypto.css:107]: I don't mind either, you decide.

Comment (astorije) [reply crypto.css:107]: Heh, whoever takes over the ownership of this repo can improve it, I'll just fix bugs I introduce lol.

**Pull request merged**